modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeSourceProcessorTest.java

```diff
@@ -23,6 +23,11 @@ public void testGradleUpgradeReleaseDxpCheck() throws Exception {
 		test("upgrade/GradleUpgradeReleaseDxpCheck.testgradle");
 	}
 
+	@Test
+	public void testJSONUpgradeLiferayThemePackageJSONCheck() throws Exception {
+		test("upgrade/LPS-195684/package.testjson");
```

Comment:
@nicolas-sss, can you please create a folder using the SF check name instead of the LPS ticket for now?